scripts/performance.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
```

Comment:
Can we use a Makefile target instead like the other tests?

Reply:
If just a `go test` it would for sure be a good Makefile target. But performance test will have some setup steps for mako, which will be added by @stati0n in the upcoming PR. I'm afraid that the logic of mako setup will be too large to be fit in a Makefile

Reply:
Let's assume [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it) for the moment and correct it in a future PR if we need to.

Reply:
Good idea, done